CONTRIBUTING.md

```diff
@@ -64,3 +64,55 @@ Other Tips
     your existing commit message by running:
 
         $ git commit --amend
+
+Checklist for Contributed Code
+------------------------------
+
+A pull request for a new feature will be reviewed to see if it meets the following requirements.  For any pull request, an astropy maintainer can help to make sure that the pull request meets the requirements for inclusion in the package.  
+
+**Scientific Quality**
```

Comment:
I've added a 'when applicable'.   As for the relevance, I would hope it would be common sense and it shouldn't be a very high hurdle to overcome, but it has come up with submissions to the stats package, for example.